FluentUI.Demo/build.gradle

```diff
@@ -64,6 +64,7 @@ dependencies {
     implementation project(':fluentui_transients')
     implementation project(':fluentui_menus')
     implementation project(':fluentui_persona')
+    implementation project(':fluentui_progress')
```

Comment:
We need to add this module in the Fluent UI container module gradle too

Reply:
added to container gradle